domdiv/__init__.py

```diff
@@ -175,8 +205,10 @@ def parse_opts(arglist):
         default=-1,
         help="stop generating after this many pages, -1 for all")
     parser.add_argument("--language",
-                        default='en_us',
-                        help="language of card texts")
+                        default=LANGUAGE_DEFAULT,
```

Comment:
I'll fix this.  Duh (hand plant to forehead.) That was the intent of automatically generating LANGUAGE_CHOICES.  Just forgot to add the important part.